### PR TITLE
docs: clarify the difference between ProcessOut and Smart Router

### PR DESCRIPTION
**Pull request by andrew-stark-cko**
## Description

The main promotional website refers to Telescope and Smart Router as the two products offered in the ProcessOut "family". The docs hardly mention Telescope at all and generally refer to the whole product as just ProcessOut. It would be useful to refer more precisely to ProcessOut (the company/service overall), Smart Router (the payment API) and Telescope (the analytical tool) as separate things in the docs.

## Solution

Rewrite the existing Getting Started page to clarify exactly what Telescope and Smart Router are. In the rest of the docs, replace any references to "ProcessOut" with "Smart Router" when they are specifically referring to the payment API. Places where the text refers to the company or the service in general should either be left as "ProcessOut" or deleted if they are used redundantly.

We also need to add a page/section in the docs for Telescope but that is a job for a future PR.

## Notes

I've also rewritten the short Migration and Test Environment pages as part of this PR and I've fixed a few small glitches and spelling mistakes as I've found them. However, I *didn't* intend to rewrite or thoroughly review every page where I've changed the ProcessOut name to Smart Router. I will certainly review the whole text as part of the project but for this PR, clarifying the names of the products is the main goal.

If it turns out that we do want the payment API to be known as "ProcessOut" after all then I'll change it back. However, it is still useful to distinguish the product name from the company name in the Markdown. We might look at placing special span elements around these names so that we can easily locate them in the future.

### CODE REVIEW

File: source/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+ProcessOut provides two main products to help merchants avoid losing
+revenue from these problems:
+
+-   **Telescope**: This tool analyzes transaction data to help merchants
+    understand which of their payments typically fail. It
+    gives them personalized advice about how to fix the problems
+    so they can complete similar payments successfully in the future.
+
+-   **Smart Router**: This is a payment API for web and mobile apps
```
Comment (louis-paul-dareau-cko) [inline source/getting-started.md:23]: (minor) Maybe worth mentioning that we also have a flexible rule system, where merchants have very granular control over their transaction routing (e.g. route based on channel, geography, card type…).

File: source/testing.md.erb [hunk 2]
```diff
@@ -4,21 +4,26 @@ title: Test environment
...
-`true`.
-
-To enable the testing environment, simply prepend your project ID with `test-`
-when instanciating the ProcessOut clients and use your sandbox private key, like so:
+It is very important to test your Smart Router code during development.
+We provide a testing environment that simulates the normal Smart Router
+production environment using mock data. This data doesn't belong
+to any real-world customers and is kept completely separate from any genuine
+data. We strongly recommend that you use the test environment during
+development to avoid the risk of damaging real customer data .
```
Comment (louis-paul-dareau-cko) [inline source/testing.md.erb:12]:
```suggestion
development to avoid the risk of damaging real customer data.
```

File: source/testing.md.erb [hunk 3]
```diff
@@ -4,21 +4,26 @@ title: Test environment
...
+
+## Using the environment
+
+You can use most of the Smart Router API [classes](/refs/) in either
+the test environment or the production environment. Note that the
+two environments are separate and so objects created in one
+environment will not be available in the other. All of the main classes
```
Comment (louis-paul-dareau-cko) [inline source/testing.md.erb:19]: (minor) I believe this is the naming we usually use when talking to merchants:

```suggestion
environment will not be available in the other. All of the main objects
```

File: source/payments/alternative-payment-methods-tokenize.md.erb [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
-title: Tokenize an alternative payment method like PayPal
+title: Tokenize an alternative payment method
 ---
 
 # Tokenize an alternative payment method
 
-Additionnally to simply accepting payments through alternative methods— such as
-local ones as iDeal, Oxxo or Alipay— ProcessOut also allows you to tokenize
+As well as simply accepting payments through alternative methods— such as
+local ones as iDeal, Oxxo or Alipay— Smart Router also allows you to tokenize
```
Comment (louis-paul-dareau-cko) [inline source/payments/alternative-payment-methods-tokenize.md.erb:8]: (minor) Not sure what your preferred style is but I personally just follow the [Practical Typography recommendations](https://practicaltypography.com/hyphens-and-dashes.html) for spacing around em dashes.

```suggestion
local ones as iDeal, Oxxo or Alipay—Smart Router also allows you to tokenize
```

File: source/migrating-to-processout.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+    Typically, we give the old PSPs access to a
+    [public key](https://en.wikipedia.org/wiki/Public-key_cryptography)
+    that they use to encrypt the exported card data. They
+    then upload the data to a secure server and grant us access.
+    Finally, we transfer the data to our own servers and decrypt
+    it.
+
+3.  **Let ProcessOut import the card data**<br/>
+    We import the card data into into our vaults as soon as it is
```
Comment (louis-paul-dareau-cko) [inline source/migrating-to-processout.md:62]:
```suggestion
    We import the card data into into our vault as soon as it is
```

**Review by louis-paul-dareau-cko — COMMENTED** [review]
Amazing work, just left a couple comments 💯

File: source/getting-started.md (hunk 1 above)
Comment (andrew-stark-cko) [reply source/getting-started.md:23]: I've added a sentence about this now.

File: source/testing.md.erb (hunk 2 above)
Comment (andrew-stark-cko) [reply source/testing.md.erb]: Well spotted! Fixed.

File: source/testing.md.erb (hunk 3 above)
Comment (andrew-stark-cko) [reply source/testing.md.erb]: Fixed.

File: source/payments/alternative-payment-methods-tokenize.md.erb (hunk 4 above)
Comment (andrew-stark-cko) [reply source/payments/alternative-payment-methods-tokenize.md.erb]: Yeah, this was just a mistake when I changed the name. Now fixed.

File: source/migrating-to-processout.md (hunk 5 above)
Comment (andrew-stark-cko) [reply source/migrating-to-processout.md]: Fixed.

**Review by louis-paul-dareau-cko — APPROVED** [review]

**Pull request merged**